policy-engine/src/main.rs

```diff
@@ -75,6 +76,11 @@ fn main() -> Result<(), Error> {
     HttpServer::new(move || {
         App::new()
             .wrap(middleware::Compress::default())
+            .wrap(
+                Cors::default()
+                    .allowed_origin("*")
```

Comment:
I think we want [allow_any_origin](https://docs.rs/actix-cors/0.5.4/actix_cors/struct.Cors.html#method.allow_any_origin) here. See also https://docs.rs/actix-cors/0.5.4/src/actix_cors/builder.rs.html#140

Reply:
Thanks; fixed with 8e4ca504 -> 95c923cd